compiler-plugin/src/main/java/io/ballerina/lib/np/compilerplugin/PromptAsCodeCodeModificationTask.java

```diff
@@ -77,55 +98,95 @@ public class PromptAsCodeCodeModificationTask implements ModifierTask<SourceModi
     private static final Token COLON = createToken(SyntaxKind.COLON_TOKEN);
     private static final Token RIGHT_DOUBLE_ARROW = createToken(SyntaxKind.RIGHT_DOUBLE_ARROW_TOKEN);
     private static final Token COMMA = createToken(SyntaxKind.COMMA_TOKEN);
+    private static final String SCHEMA_ANNOTATION_IDENTIFIER = "Schema";
+    private static final String CALL_LLM = "callLlm";
+    private static final String STRING = "string";
+    private static final String BYTE = "byte";
+    private static final String NUMBER = "number";
 
+    private static Optional<String> npPrefixIfImported = Optional.empty();
```

Comment:
The reason for moving this to the global level is that it is needed for both the function modifier and the type definition modifier. In the future, if we introduce new modifiers, this will still be required.
https://github.com/ballerina-platform/module-ballerinax-np/pull/11#discussion_r1984554105

Reply:
I would avoid class level state and pass it as an argument where necessary.

Reply:
Fixed.